etl/steps/data/garden/unodc/2025-05-12/justice.meta.yml

```diff
@@ -0,0 +1,46 @@
+# NOTE: To learn more about the fields, hover over their names.
+definitions:
+  common:
+    description_key:
+      - Crime and justice data can be affected by many factors, such as how a country’s justice system works, whether the data follow the “principal offence rule” (which means if a person commits more than one crime, it only counts once), how complete the data are, and how closely a country follows international standards for reporting.
+      - Even though these differences exist, the data are still extremely useful for understanding how criminal justice systems work in different countries.
+      - The data are collected from official records kept by different parts of the justice system. The police usually record information about people who are suspected of, arrested for, or cautioned for a crime. Prosecutors collect data about people being charged, and the courts collect information about people who appear before them and those who are convicted.
+      - The United Nations gathers these data each year through a survey called the United Nations Survey of Crime Trends and Operations of Criminal Justice Systems (UN-CTS). Experts in each country, who are appointed by their government, collect and combine the data from all relevant agencies before sending it to the UN.
+      - The UN Office on Drugs and Crime (UNODC) checks the data for accuracy and to make sure it is consistent with other sources.
```

Comment:
The last four points do not look that important to me. I assume they are part of the processing of any type of data.